Assets/MRTK/SDK/Profiles/DefaultMixedRealityXRSDKControllerVisualizationProfile.asset

```diff
@@ -51,23 +51,47 @@ MonoBehaviour:
     controllerVisualizationType:
       reference: Microsoft.MixedReality.Toolkit.Input.WindowsMixedRealityControllerVisualizer,
         Microsoft.MixedReality.Toolkit.SDK
-  - description: Oculus Hand
+  - description: Windows Mixed Reality Articulated Hand
     controllerType:
-      reference: Microsoft.MixedReality.Toolkit.XRSDK.Oculus.Input.OculusHand, Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus
+      reference: Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityArticulatedHand,
```

Comment:
Oh i guess actually, this isn't needed on the XR SDK profile. If we aren't making any changes to the legacy default profile, I think the below `WindowsMixedRealityXRSDKArticulatedHand` are the only ones we need to care about

Reply:
I had forgotten this was focused on XR SDK

Reply:
I included changes to the default Controller visualization profile as well since it's what's loaded by default, and is also  the profile that is initially loaded when you open the hand interaction scene

Reply:
Oh right! I see now. Okay cool, so I think we'd only need `WindowsMixedRealityXRSDKArticulatedHand` in the XR SDK one and `WindowsMixedRealityArticulatedHand` in the legacy one, instead of both in both.